cgmanifest.json

```diff
@@ -1142,16 +1142,6 @@
         }
       }
     },
-    {
```

Comment:
Entry for containernetworking-plugin should also be removed

Reply:
Great catch!  Fixed